ts/components/conversation/ConversationHeader.tsx

```diff
@@ -575,13 +579,16 @@ export class ConversationHeader extends React.Component<PropsType, StateType> {
     }
 
     const contents = (
-      <>
+      <div
+        className="module-conversation-header__title-container"
```

Comment:
Could this be `ConversationHeader` instead of `conversation-header` to match the rest of the file?

```suggestion
        className="module-ConversationHeader__title-container"
```

Reply:
Nice spot. This must've been legacy from when I first did the commits